apps/settings/js/modules/wifi_utils.js

```diff
@@ -250,6 +253,57 @@ define(function(require) {
       icon.className = icon.className.replace(/level-\w*/, '');
       var level = Math.min(Math.floor(networkSignal / 20), 4);
       icon.classList.add('level-' + level);
+    },
+
+    getNetworkKey: function(network) {
+      if (!network) {
+        return '';
+      } else {
+        var key =
+          network.ssid + '+' + WifiHelper.getSecurity(network).join('+');
+        return key;
+      }
+    },
+
+    updateListItemStatus: function(options) {
+      var listItems = options.listItems;
+      var activeItem = options.activeItem;
+      var event = options.event;
+
+      if (!event || !listItems) {
```

Comment:
Here, we can do early return to gracefully degrade if needed elements are not passing in and would use `console.log` to print some log informations.